### ISSUE TITLE
Mobile web: Swipe up on full screen media to open media from another post

### ISSUE THREAD

**Issue by aquariusdev** [gabboman/wafrn#45]
- After opening media (images, video, galleries) in the full screen lightbox display, a user should be able to swipe up to open another media in full screen from users, hashtags or lists one follows.
- Media in full screen would, at the bottom, show the user, a follow button, first line from any text in the container post, a "more" link to open the main post, and post action buttons (fave, reshare/quote, bookmark, reply).
- Alt text would display above the post summary text
- Swiping up may either pull from the latest posts or random posts from the aforementioned sources.

This can work like how Tumblr's app currently works. This should also be an optional setting which can be turned on/off in settings.

**Comment by gabboman** [comment]
Hi, this is a feature request right?

This one is a non trivial change, but I will look into it once I finish the migration from primeng to angular material

**Comment by aquariusdev** [comment]
Yes this is a feature request. I thought I had labeled it but apparently I didn't? Weird.

**Comment by gabboman** [comment]
this feature is now kinda possible with the new angular material frontend, will look into it in the future. I leave this as an todo

**Comment by gabboman** [comment]
done in #224 by @FireIsGood
